src/main/java/uk/gov/digital/ho/hocs/workflow/api/FormService.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.digital.ho.hocs.workflow.api;
+
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.workflow.api.dto.GetFormResponse;
+import uk.gov.digital.ho.hocs.workflow.api.dto.SchemaDto;
+import uk.gov.digital.ho.hocs.workflow.client.caseworkclient.CaseworkClient;
+import uk.gov.digital.ho.hocs.workflow.client.caseworkclient.dto.GetCaseworkCaseDataResponse;
+import uk.gov.digital.ho.hocs.workflow.client.infoclient.InfoClient;
+import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsForm;
+import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsFormAccordion;
+import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsFormField;
+import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsFormSecondaryAction;
+import uk.gov.digital.ho.hocs.workflow.domain.model.forms.HocsSchema;
+
+import java.util.List;
+
+import java.util.UUID;
+
+@Service("HocsFormService")
```

Comment:
Camunda registers a bean of FormService, this allows us to keep the sane name but just flag this in Spring land with this name.